locksmith/src/operations/userOperations.ts

```diff
@@ -0,0 +1,41 @@
+const ethJsUtil = require('ethereumjs-util')
+const models = require('../models')
+const { User, UserReference } = models
+import * as types from '../types'
+
+class UserOperations {
+  static async createUser(input: types.UserCreationInput): Promise<Boolean> {
+    let user: any = await User.create({
+      publicKey: ethJsUtil.toChecksumAddress(input.publicKey),
+      recoverySeed: input.recoverySeed,
+      encryptedPrivateKey: input.encryptedPrivateKey,
+    })
+
+    let userReference = await UserReference.create({
+      emailAddress: input.emailAddress.toLowerCase(),
+    })
+
+    if (userReference) {
+      userReference.setUser(user)
+      userReference.save()
+      return true
+    } else {
+      return false
+    }
+  }
+
+  static async privateKeyLookup(emailAddress: String): Promise<String | null> {
```

Comment:
`getUserPrivateKeyByEmailAddress` would probably be more consistent with how we named other functions?